lib/lambda_ethereum_consensus/p2p/gossip/blob_sidecar.ex

```diff
@@ -72,7 +24,38 @@ defmodule LambdaEthereumConsensus.P2P.Gossip.BlobSideCar do
         Logger.warning("[Gossip] Blob rejected, reason: #{inspect(reason)}")
         Libp2pPort.validate_message(msg_id, :reject)
     end
+  end
+
+  @spec join_topics() :: :ok
+  def join_topics() do
+    topics()
+    |> Enum.each(fn topic_name -> Libp2pPort.join_topic(self(), topic_name) end)
```

Comment:
We should not use `each` if it can return an error and leave the node in an unusable state. We should be able to return an error here, for example, with a reduce, or with a map and then counting the errors.